docs/source/gpu.rst

```diff
@@ -0,0 +1,156 @@
+            height="315"
+            src="https://developer.download.nvidia.com/video/gputechconf/gtc/2019/video/S9198/s9198-dask-and-v100s-for-fast-distributed-batch-scoring-of-computer-vision-workloads.mp4"
+            frameborder="0"
+            allow="autoplay; encrypted-media"
+            allowfullscreen>
+    </iframe>
+
+
+
+High Level Collections
+----------------------
+
+Dask can also help to scale out large scale array and data computations by
+combining the Dask collections Dask Array and Dask DataFrame, with a
+GPU-accelerated array or dataframe library.
+
+Recall that :doc:`Dask Array <array>` creates a large array out of many NumPy
+arrays and :doc:`Dask DataFrame <dataframe>` creates a large dataframe out of
+many Pandas dataframes.  We can use these same systems with GPUs if we swap out
+the NumPy/Pandas components with GPU-accelerated versions of those same
+libraries, as long as the GPU accelerated version looks enough like
+NumPy/Pandas in order to interoperate with Dask.
+
+Fortunately, libraries that mimic NumPy, Pandas, and Scikit-Learn on the GPU do
+exist.
+
+
+DataFrames
+~~~~~~~~~~
+
+The `RAPIDS <https://rapids.ai>`_ libraries provide a GPU accelerated
+Pandas-like library,
+`cuDF <https://rapidsai.github.io/projects/cudf/en/latest/>`_,
+which interoperates well and is tested against Dask DataFrame.
+
+If you have cudf installed then you should be able to convert a Pandas-backed
+Dask DataFrame to a cuDF-backed Dask DataFrame as follows:
+
+.. code-block:: python
+
+   import cudf
+
+   df = df.map_partitions(cudf.from_pandas)  # convert pandas partitions into cudf partitions
+
+However, cuDF does not support the entire Pandas interface, and so a variety of
+Dask DataFrame operations will not function properly.
+
+
+Arrays
+~~~~~~
+
+.. note:: Dask's integration with CuPy relies on features recently added to
+   NumPy and CuPy, particularly in version ``numpy>=1.17`` and ``cupy>=6``
+
+`Chainer's CuPy <https://cupy.chainer.org/>`_ library provides a GPU
+accelerated NumPy-like library that interoperates nicely with Dask Array.
+
+If you have a cupy installed then you should be able to convert a NumPy-backed
+Dask Array into a CuPy backed Dask Array as follows:
+
+.. code-block:: python
+
+   import cupy
+
+   x = x.map_blocks(cupy.asarray)
+
+CuPy is fairly mature and adheres closely to the NumPy API.  However, small
+differences do exist and these can cause Dask Array operations to function
+improperly.
+
+
+Scikit-Learn
+~~~~~~~~~~~~
+
+There are a variety of GPU accelerated machine learning libraries that follow
+the Scikit-Learn Estimator API of fit, transform, and predict.  These can
+generally be used within `Dask-ML's <https://ml.dask.org>`_ meta estimators,
+such as `hyper parameter optimization <https://ml.dask.org/hyper-parameter-search.html>`_.
+
+Some of these include:
+
+-  `Skorch <https://skorch.readthedocs.io/>`_
+-  `cuML <https://rapidsai.github.io/projects/cuml/en/latest/>`_
+-  TODO (help?)
```

Comment:
You could also include 
- https://github.com/Xtra-Computing/thundergbm
- https://github.com/Xtra-Computing/thundersvm
- https://github.com/Microsoft/LightGBM
- xgboost ?